app/next-client-app/components/homepage/CTA.tsx

```diff
@@ -0,0 +1,46 @@
+import BoxReveal from "@/components/magicui/box-reveal";
+import { Button } from "@/components/ui/button";
+import { Github, Rocket } from "lucide-react";
+import Link from "next/link";
+
+export default function CallToAction() {
+  const deployLink = process.env.DEPLOY_LINK ?? "";
+  return (
+    <>
+      {/* CTA section */}
+      <div className="flex flex-col items-center text-center px-5 lg:px-0">
+        <BoxReveal boxColor={"#e6a312"} duration={1.0}>
+          <h1 className="sm:text-5xl text-pretty text-3xl">
+            Join our community
+          </h1>
+        </BoxReveal>
+        <BoxReveal boxColor={"#e6a312"} duration={1.0}>
+          <h1 className="opacity-80 sm:text-2xl text-pretty text-md my-4">
+            Start a Github discussion{" "}
+            {deployLink !== "" && <span>deploy your own Carrot</span>} today!
+          </h1>
+        </BoxReveal>
+        <BoxReveal boxColor={"#e6a312"} duration={1.0}>
+          <div className="flex gap-5 lg:flex-row flex-col">
+            <Link
+              href={
+                "https://github.com/Health-Informatics-UoN/Carrot-Mapper/discussions"
```

Comment:
I think using `<a>` for linking out in Next - we would also like this to open in a new tab.